packages/css-library/compile-stylesheets.js

```diff
@@ -0,0 +1,29 @@
+const { exec } = require('child_process');
+
+exec("cp src/stylesheets/_mixins.scss dist/stylesheets/_mixins.scss");
+exec("sass --load-path=../../node_modules/@uswds/uswds/packages/ src/stylesheets/_core.scss:dist/stylesheets/_core.css");
```

Comment:
Is this needed because we are trying to keep the underscore `_` in the path name?